Main/src/main/java/com/main/acad/dao/ChapterDao.java

```diff
@@ -0,0 +1,104 @@
+package main.java.com.main.acad.dao;
+
+import main.java.com.main.acad.entity.Chapter;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class ChapterDao implements Dao {
```

Comment:
Why these changes in your pull request? This mean that you not branched from latest master.